ui/src/modules/autoconsent/views/home.js

```diff
@@ -47,7 +51,7 @@ export default define({
                 name="scope"
                 value="selected"
                 onchange="${html.set('scope')}"
-                checked
+                checked="${scope === 'selected'}"
```

Comment:
Please test it, as it works slightly different than your code. If it overwrites the value form uncontrolled input, we can use `defaultChecked`.